src/components/mgt-person/mgt-person.ts

```diff
@@ -445,6 +444,35 @@ export class MgtPerson extends MgtTemplatedComponent {
       } else {
         const initials = this.getInitials();
 
+        // work around to import fabric ui core colors without creating .d.ts file
+        const colors = [
+          'pinkRed10',
+          'red20',
+          'red10',
+          'orange20',
+          'orangeYellow20',
+          'green10',
+          'green20',
+          'cyan20',
+          'cyan30',
+          'cyanBlue10',
+          'cyanBlue20',
+          'blue10',
+          'blueMagenta30',
+          'blueMagenta20',
+          'magenta20',
+          'magenta10',
+          'magentaPink10',
+          'orange30',
+          'gray30',
+          'gray20'
+        ];
+        const initialsList = this.renderRoot.querySelectorAll('.initials') as NodeListOf<HTMLElement>;
+
+        initialsList.forEach(i => {
+          i.classList.add(colors[this.numberFromInitials(initials) % colors.length]);
+        });
```

Comment:
instead of using a queryselector in the render method (which is not recommended), the class should be added along side the other classes in `imageClasses`.

ex:
```ts
imageClasses[colors[this.numberFromInitials(initials) % colors.length]] = true;
```

See [classMap](https://lit-html.polymer-project.org/guide/template-reference#classmap) for reference on adding classes with lit-html.

Reply:
done